setup.py

```diff
@@ -120,11 +120,11 @@ def get_git_sha() -> str:
         "sqlparse>=0.4.3, <0.5",
         "tabulate>=0.8.9, <0.9",
         # needed to support Literal (3.8) and TypeGuard (3.10)
-        "typing-extensions>=3.10, <4",
+        "typing-extensions>=4, <5",
```

Comment:
Can we add a comment to the description that `typing-extensions` is also bumped in this PR? Also, can you remove the reference to `Literal` on line 122, as we're no longer supporting Python 3.7 or lower (in other words, we currently only need `TypeGuard` from `typing-extensions`)

Reply:
sure thing!